vod_transcode.sh

```diff
@@ -9,6 +9,9 @@
 #  sudo service antmedia restart
 #
 
+# Just convert to HLS 
```

Comment:
HLS variable and bitrates create some kind of ambiguity.  I mean let the specify the bitrates and if there is no bitrate, it's just HLS conversion